### PR TITLE
Persia + euro ameriga

### PR DESCRIPTION
**Pull request by Kuri21079**
Amazonie starts out as the only independent democracy on the continent, essentially buying out its own independence from the Dual Monarchy while allowing for elections. However, it is still divided over its internal debate regarding slavery.
 
- Equinoxea is now playable from the start as a puppet of Amazonie
- Once the starting truce between Equinoxea and Amazonie expires, there will be a series of journal entries and events regarding the slavery debate
- Amazonie has the potential to have an additional civil war, depending on which interest group it alienates 
- Equinoxea can get the ability to claim all Afro-Amerigan homelands if it researches pan-nationalism
- Other Francophone nations in Ameriga can have the potential get Afro-Amerigan as a primary culture, with the exception of Antarctica. Amazonie can go through this route too, but only by event from journal entries.

Meanwhile, Beornia is a relatively poor and underdeveloped nation. Having recently returned from a trip to Europe, its current king, Edward, hopes to industrialize the nation and bring it closer to the rest of the world while his uncle, Charles, believes Edward is compromising the country's character.

- Beornia starts out with a journal entry regarding the influence of its landowners, the "Cattle Barons" and their conflict with the country's less influential industrialists, the "Edwardian Circle". 
- If Scandinavia is not doing well economically, Beornia can purchase the Falklands from them.
- Forming the Holy Britannian Empire is now a decision that requires Beornia to be a major power with certain laws rather than a conventional formable.
- The Holy Britannian Empire has a journal entry and special event about conquering the rest of Ameriga.
- If pan-nationalism is researched, Beornia has an event chain on merging with England in some way if both are independent and have good relations.
- Beornia also has the potential to claim all of England if relations end up being poor instead.
- If both end up as monarchies, Lotharingia and Beornia can unite through decision.

Persia/Central Asia

In Persia, the Timurtash Empire is in decline with different breakaway states vying for power. The Kar-Kiya, Sheikhbizin, Shaki, and Karlani contest one another for hegemony in the region, though they all have their own internal issues in their respective journal entries.
   - Kar-Kiya has a journal entry/event chain regarding its interest groups and standard of living.
   - Shaki Khanate starts out with two Georgian puppets they can annex later
   - Shaki has to deal with Armenian separatists if they continue to be discriminated against. 
   - Caucasia and Turkestan have been added as formables for their respective cultures. Iraq and Afghanistan can also form Persia.
   - If Armenia breaks free and is no longer an insignificant power, they can claim all Armenian homelands.
   - Sorani can add Assyrians as a primary culture. They also start out with cultural exclusion.
   - Near the start of the game, Sheikhbizin can get a research boost if they let Spain establish a port in the area.
   - Sheikhbizin gets a JE regarding the nomads in the country along a new culture, "Qashqai-Lurish" that is both Iranic and Turkish.
   - Timurtash can get a research boost if they embrace the Ismailis, which requires enfranchising them.
   - Central Asian cultural traits have been reworked.
   - Dzungar Khanate has the potential to collapse if certain conditions are met.

### CODE REVIEW

File: events/dvg_amazonie.txt [hunk 1]
```diff
@@ -0,0 +1,2102 @@
...
+					type = annex_country
+					target_country = c:AMA
+				}
+			}			
+	    }
+		play_as = c:ATL
+	}
+
+	option = {
+		name = dvg_amazonie.13.aa
```
Comment (MadJamez) [inline events/dvg_amazonie.txt:1340]: Unrecognized loc key dvg_amazonie.13.aa

File: events/dvg_england.txt [hunk 2]
```diff
@@ -0,0 +1,776 @@
...
+		default_option = yes
+		
+		c:ENG = {
+			create_diplomatic_pact = {
+				country = c:BEO
+				type = customs_union
+			}
+		
+		
+			ai_chance = {
```
Comment (MadJamez) [inline events/dvg_england.txt:435]: Unknown effect ai_chance at  file: events/dvg_england.txt line: 435
It is at the wrong level here. Should be outside the c:ENG tag

File: common/history/characters/0_dvg_shekhbizin .txt [hunk 3]
```diff
@@ -0,0 +1,35 @@
+﻿CHARACTERS = {
+	c:SHZ = {
+	    create_character = {
```
Comment (MadJamez) [inline common/history/characters/0_dvg_shekhbizin .txt:3]: The ruler of Shekhbizin Empire is scripted to be Sadeq Shekhbizin but that character leads the interest group of Qashqai Confederacy that they belong to, which is not allowed under Dictatorial/Hereditary

**Review by MadJamez — CHANGES_REQUESTED** [review]

File: common/journal_entries/dvg_dzungaria.txt [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+		}
+	}
+
+	on_monthly_pulse = {
+		effect = {
+			if = {
+				limit = {
+					exists = c:DZU
+					c:DZU = ROOT
+					ruler = { has_variable = is_maximilian_jagiello }
```
Comment (MadJamez) [inline common/journal_entries/dvg_dzungaria.txt:40]: Should be is_galdan_khan

File: common/journal_entries/dvg_dzungaria.txt [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+					year >= 1846
+				}
+				ruler = { kill_character = yes }
+			}
+			if = {
+				limit = {
+					exists = c:DZU
+					c:DZU = ROOT
+					ruler = { has_variable = is_maximilian_jagiello }
```
Comment (MadJamez) [inline common/journal_entries/dvg_dzungaria.txt:49]: Should be is_galdan_khan

File: common/journal_entries/dvg_dzungaria.txt [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+	}
+
+	on_monthly_pulse = {
+		effect = {
+			if = {
+				limit = {
+					exists = c:DZU
+					c:DZU = ROOT
+					ruler = { has_variable = is_maximilian_jagiello }
+					year >= 1846
```
Comment (MadJamez) [inline common/journal_entries/dvg_dzungaria.txt:41]: Make this 1855 or something?

File: common/journal_entries/dvg_dzungaria.txt [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+				}
+				ruler = { kill_character = yes }
+			}
+			if = {
+				limit = {
+					exists = c:DZU
+					c:DZU = ROOT
+					ruler = { has_variable = is_maximilian_jagiello }
+					year >= 1837
```
Comment (MadJamez) [inline common/journal_entries/dvg_dzungaria.txt:50]: Make this 1850 or something to give players more time.

**Review by MadJamez — CHANGES_REQUESTED** [review]

**Review by MadJamez — CHANGES_REQUESTED** [review]
17:25:51][jomini_trigger.cpp:1252]: Left and right side do not have matching types. Left has type(s) 'country', right has type(s) 'value' at ' file: events/dvg_england.txt line: 239'
[17:25:51][jomini_effect.cpp:793]: Inconsistent effect scopes (country vs. state) in file: events/dvg_england.txt line: 612

[17:25:58][jomini_script_system.cpp:263]: Script system error!
  Error: set_owner_of_provinces effect [ Province 'x809040' crossing multiple state regions ]
  Script location: file: events/dvg_persia.txt line: 127

[17:25:58][character_templates_utils.cpp:29]: Invalid Character trait provided offensive_planner in  file: common/history/characters/0_dvg_timurtash.txt line: 20
[17:25:58][jomini_effect.cpp:128]: PostValidate of effect 'create_character' returned false at  file: common/history/characters/0_dvg_timurtash.txt line: 20

**Comment by Kuri21079** [comment]
Not sure what can be done about "Province 'x809040' crossing multiple state regions". Searching the province id in this repository doesn't show it repeating but that it's only a part of Azerbaijan. The event for it still fires accordingly with the province being part of breakaway Armenia.

**Comment by MadJamez** [comment]
> x809040

Maybe because not all provinces there are in the same state? Might just be an error because of that, with no real issue at the moment.

File: events/dvg_england.txt [hunk 8]
```diff
@@ -0,0 +1,786 @@
+﻿namespace = dvg_england
```
Comment (MadJamez) [inline events/dvg_england.txt:1]: Left and right side do not have matching types. Left has type(s) 'country', right has type(s) 'value' at ' file: events/dvg_england.txt line: 239'

**Review by MadJamez — CHANGES_REQUESTED** [review]

**Comment by MadJamez** [comment]
@Kuri21079 Can you check if all characters you made immortal also die in every path? If not maybe remove immortality trait after a certain year for them, or through journal?

**Review by MadJamez — DISMISSED** [review]

File: .metadata/metadata.json [hunk 9]
```diff
@@ -1,5 +1,9 @@
 {
-	"name" : "Divergences (Dev)",
+
+
+	"name" : "Divergences (PRS)",
```
Comment (MadJamez) [inline .metadata/metadata.json:4]: Can you change this back to DEV? Otherwise this will get merged in main.

**Review by MadJamez — APPROVED** [review]

**Pull request merged**